storage/streams/src/main/java/io/apicurio/registry/streams/StreamsRegistryConfiguration.java

```diff
@@ -112,11 +114,15 @@ public void stopStorageProducer(@Disposes ProducerActions<Str.ArtifactKey, Str.S
     @Produces
     @Singleton // required (cannot be ApplicationScoped), as we don't want proxy
     public KafkaClientSupplier kafkaClientSupplier(StreamsProperties properties) {
+        log.debug("Providing a new KafkaClientSupplier instance.");
         KafkaClientSupplier kcs = new DefaultKafkaClientSupplier();
         if (!properties.ignoreAutoCreate()) {
             Map<String, Object> configMap = new HashMap(properties.getProperties());
             try (Admin admin = kcs.getAdmin(configMap)) {
-                KafkaUtil.createTopics(admin, Collections.singleton(properties.getStorageTopic()));
+                Set<String> topicNames = new LinkedHashSet<>();
+                topicNames.add(properties.getStorageTopic());
+                topicNames.add(properties.getGlobalIdTopic());
+                KafkaUtil.createTopics(admin, topicNames);
```

Comment:
@jsenko @carlesarnal @famartinrh What properties should these topics have?  Does the operator create these topics?  I know for Kafka+SQL I'm auto-creating the topic with log-compaction enabled.  Are there any default options we need to set that we're not setting?

Reply:
If they do not exist, we're creating them using `KafkaUtil`. There are some defaults there that are being used for the streams storage since we're sending a null config. Those defaults match the expected configuration.